CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * Add an explicit margin zero to the super navigation mobile menu button ([PR #2445](https://github.com/alphagov/govuk_publishing_components/pull/2445))
 * Remove brexit as a topic from the super navigation header ([PR #2446](https://github.com/alphagov/govuk_publishing_components/pull/2446))
+* Update the pseudo underline mixin and it's usage on the super navigation header ([PR #2439](https://github.com/alphagov/govuk_publishing_components/pull/2439))
```

Comment:
```suggestion
* Update the pseudo underline mixin and its usage on the super navigation header ([PR #2439](https://github.com/alphagov/govuk_publishing_components/pull/2439))
```

Reply:
Ooooops. I'll try to fix this in another PR or a release.

Reply:
Bother, I thought I'd submitted this in time. Ah well, never mind - it's not a code error and easily fixed.